src/popup/components/Layout/Header/index.tsx

```diff
@@ -32,8 +33,8 @@ type HeaderProps = {
   className?: string;
 };
 
-export const Header = ({ className }: HeaderProps) => (
-  <HeaderEl className={className}>
+export const Header = ({ className, ...props }: HeaderProps) => (
```

Comment:
Added `...props` to follow our [product conventions](https://github.com/stellar/product-conventions/blob/master/STYLEGUIDE.md#pass-props)